tests/client/api_events/test_outputs_payloads.c

```diff
@@ -5,50 +5,27 @@
 #include <time.h>
 
 #include "client/api/events/sub_outputs_payload.h"
+#include "client/api/restful/get_output.h"
 #include "test_config.h"
 #include "unity/unity.h"
 
 bool test_completed = false;
 char const *const test_bech32 = "atoi1qqs7y6ec5vcg6cnz46vjrar2epc52lhksyar3a4zua7fg7ca08y5ymep8aa";
-char const *const test_ed25519 = "21e26b38a3308d6262ae9921f46ac871457ef6813a38f6a2e77c947b1d79c942";
-char const *const test_output_id = "3912942d1cb588d8091eff2069bdd797a0a834739dc8ea550e35fb0dc8609c820000";
+char const *const test_output_id = "5e753f69b44870aa6a90adf2c366dccac00097c41d5c884dd81ef7cf29eefdd70000";
+char alias_id[] = "23dc192ede262b3f4bce379b26c31bad029f62fe";
+char nft_id[] = "efdc112efe262b304bcf379b26c31bad029f61de";
+char foundry_id[] = "56ec192ede262b3f4bce379b26c31bad029f63bc23ef56ee48cf";
```

Comment:
those are const strings, for consistency
```suggestion
char const *const alias_id = "23dc192ede262b3f4bce379b26c31bad029f62fe";
char const *const nft_id = "efdc112efe262b304bcf379b26c31bad029f61de";
char const *const foundry_id = "56ec192ede262b3f4bce379b26c31bad029f63bc23ef56ee48cf";
```